<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NavigationDockWidget</class>
 <widget class="QDockWidget" name="NavigationDockWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>139</width>
    <height>564</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowIcon">
   <iconset resource="../../../../Resources/axes.qrc">
    <normaloff>:/fig/navigation.png</normaloff>:/fig/navigation.png</iconset>
  </property>
  <property name="allowedAreas">
   <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
  </property>
  <property name="windowTitle">
   <string>Navigation</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0" colspan="2">
     <layout class="QGridLayout" name="gridLayout" rowminimumheight="10,10,10,10,0,0,0,0,0,0,0,0">
      <property name="sizeConstraint">
       <enum>QLayout::SetMinimumSize</enum>
      </property>
      <property name="spacing">
       <number>6</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="XSliceLbl">
        <property name="minimumSize">
         <size>
          <width>54</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>XSlice</string>
        </property>
        <property name="buddy">
         <cstring>XSliceSpinBox</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="XSliceSpinBox"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="YSliceLbl">
        <property name="text">
         <string>YSlice</string>
        </property>
        <property name="buddy">
         <cstring>YSliceSpinBox</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="YSliceSpinBox"/>
      </item>
      <item row="6" column="1">
       <widget class="QSpinBox" name="ZSliceSpinBox"/>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="ZSliceLbl">
        <property name="text">
         <string>ZSlice</string>
        </property>
        <property name="buddy">
         <cstring>ZSliceSpinBox</cstring>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QLabel" name="TSliceLbl">
        <property name="text">
         <string>TSlice</string>
        </property>
        <property name="buddy">
         <cstring>TSliceSpinBox</cstring>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <widget class="QSpinBox" name="TSliceSpinBox"/>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QSlider" name="XSliceSlider">
        <property name="value">
         <number>0</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksAbove</enum>
        </property>
        <property name="tickInterval">
         <number>100</number>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="2">
       <widget class="QSlider" name="YSliceSlider">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksAbove</enum>
        </property>
        <property name="tickInterval">
         <number>100</number>
        </property>
       </widget>
      </item>
      <item row="8" column="0" colspan="2">
       <widget class="QSlider" name="ZSliceSlider">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksAbove</enum>
        </property>
       </widget>
      </item>
      <item row="11" column="0" colspan="2">
       <widget class="QSlider" name="TSliceSlider">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksAbove</enum>
        </property>
        <property name="tickInterval">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="MinXSlicelbl">
        <property name="text">
         <string>Min</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="MaxXSlicelbl">
        <property name="text">
         <string>Max</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="MinYSlicelbl">
        <property name="text">
         <string>Min</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="MaxYSlicelbl">
        <property name="text">
         <string>Max</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="MinZSlicelbl">
        <property name="text">
         <string>Min</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QLabel" name="MaxZSlicelbl">
        <property name="text">
         <string>Max</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="10" column="0">
       <widget class="QLabel" name="MinTSlicelbl">
        <property name="text">
         <string>Min</string>
        </property>
       </widget>
      </item>
      <item row="10" column="1">
       <widget class="QLabel" name="MaxTSlicelbl">
        <property name="text">
         <string>Max</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0" colspan="2">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0" colspan="2">
       <widget class="QComboBox" name="ModeComboBox">
        <item>
         <property name="text">
          <string>Classic</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Doppler</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="stepLabel">
        <property name="text">
         <string>Step</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="step">
        <property name="value">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="tLabel">
        <property name="text">
         <string>Nb of T</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="t">
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="value">
         <number>3</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="channelLabel">
        <property name="text">
         <string>Channel</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="channelName">
        <property name="text">
         <string>Name</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="1">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>XSliceSpinBox</tabstop>
  <tabstop>YSliceSpinBox</tabstop>
  <tabstop>ZSliceSpinBox</tabstop>
  <tabstop>TSliceSpinBox</tabstop>
 </tabstops>
 <resources>
  <include location="../../../../Resources/axes.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>XSliceSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>XSliceSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>98</x>
     <y>42</y>
    </hint>
    <hint type="destinationlabel">
     <x>68</x>
     <y>69</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>YSliceSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>YSliceSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>98</x>
     <y>95</y>
    </hint>
    <hint type="destinationlabel">
     <x>68</x>
     <y>122</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ZSliceSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ZSliceSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>98</x>
     <y>148</y>
    </hint>
    <hint type="destinationlabel">
     <x>68</x>
     <y>175</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>TSliceSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>TSliceSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>98</x>
     <y>201</y>
    </hint>
    <hint type="destinationlabel">
     <x>68</x>
     <y>228</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>XSliceSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>XSliceSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>68</x>
     <y>69</y>
    </hint>
    <hint type="destinationlabel">
     <x>98</x>
     <y>42</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>YSliceSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>YSliceSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>68</x>
     <y>122</y>
    </hint>
    <hint type="destinationlabel">
     <x>98</x>
     <y>95</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ZSliceSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ZSliceSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>68</x>
     <y>175</y>
    </hint>
    <hint type="destinationlabel">
     <x>98</x>
     <y>148</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>TSliceSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>TSliceSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>68</x>
     <y>228</y>
    </hint>
    <hint type="destinationlabel">
     <x>98</x>
     <y>201</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
